services/powchain/src/main/java/tech/pegasys/teku/services/powchain/execution/ExecutionEngineChannelImpl.java

```diff
@@ -68,34 +66,24 @@ private static void printConsole(String formatString, Object... args) {
   }
 
   @Override
-  public SafeFuture<UInt64> preparePayload(
```

Comment:
seems like you're rolling back my previous PR :-)

Reply:
Yep, sorry. Sloppy merge :( 
Already fixed